geode-core/src/distributedTest/java/org/apache/geode/management/DistributedSystemMXBeanWithAlertsDistributedTest.java

```diff
@@ -137,6 +137,7 @@ public void setUp() throws Exception {
     memberVM3 = getVM(3);
 
     managerMember = managerVM.invoke(() -> createManager());
+    IgnoredException.addIgnoredException("Cannot form connection to alert listener");
```

Comment:
What is the reason for adding this as an ignored exception?  Why do we now expect to see this exception and feel okay ignoring it?

Reply:
Due to previous solution, sender thread did not detect release of connection. So this is the reason why this exception did not appear in test logs.